src/cita-sdk.js

```diff
@@ -1,18 +1,43 @@
 const { default: CITASDK } = require("@cryptape/cita-sdk");
-
+const web3 = require("web3");
 const config = require("./config");
-require('dotenv').config();
+require("dotenv").config();
 
 var cita; // config.chain indicates that the address of CITA to interact
 
-console.log("It working in " + process.env.REACT_APP_RUNTIME);
+const REACT_APP_RUNTIME = process.env.REACT_APP_RUNTIME;
+console.log("It working in " + REACT_APP_RUNTIME);
 if (process.env.REACT_APP_RUNTIME === "cyton") {
   cita = CITASDK(window.cita.currentProvider);
   cita.currentProvider.setHost(config.chain);
+} else if (process.env.REACT_APP_RUNTIME === "webRandomPk") {
```

Comment:
This is a new env called "webRandomPk", please add more description to the PR's description for what's changed in this BR, and also update the README.

Reply:
@travis-joe need to update the REAME